requirements.txt

```
@@ -1,7 +1,7 @@
 # Version updates managed by dependabot
 
 apypie==0.7.1
-broker[docker,podman,hussh]==0.6.12
+broker[ansibletower,hussh]==0.7.0rc3
```

Broker no longer has a default ssh backend installed. While I want to move to Hussh as the backend we use, there we can keep ssh2-python for a bit longer as an option. Most ci config also uses it as the backend as well.
```suggestion
broker[ansibletower,podman,hussh,ssh2_python]==0.7.0rc3
```